scheduler/src/cook/mesos/api.clj

```diff
@@ -609,6 +632,9 @@
                                uris)})
                  (when labels
                    {:labels (walk/stringify-keys labels)})
+                 (when constraints
```

Comment:
Would rather do that in a separate pr, there are a lot of these conditions that would change